app/room.py

```diff
@@ -1,19 +1,21 @@
-class Room:
+class RoomClass(object):
 
-    def __init__(self, room_name, room_type, max_occupants):
+    def __init__(self, room_name=None, room_type=None, max_occupants=None,
+                 occupants=None):
         self.room_name = room_name
         self.room_type = room_type
+        self.occupants = occupants
```

Comment:
By default during room creation, there are 0 occupants. It will be better to set `self.occupants = occupants` and avoid setting it in every subclass.